script.js

```diff
@@ -192,11 +210,17 @@ function handleData([sheetsText, statesGeoJson]) {
   // add both the states layer and localities layer to the map
   // and save the layer output
   const states = handleStatesLayer(statesGeoJson);
+  const counties = handleCountiesLayer(countiesGeoJson);
   const localities = handleLocalitiesLayer(localitiesGeoJson);
 
+  // icons for the legend / layer control
+  let cityIcon = "<svg width="+ 2.4* pointRadius +" height="+ 2.4* pointRadius +"><circle cx="+ 1.2*pointRadius +" cy="+ 1.2*pointRadius +" r=" + pointRadius + " stroke='#4dac26' stroke-width=" + strokeWeight + " fill='#b8e186' fill-opacity='" +fillOpacity + "'/></svg>"
```

Comment:
@goldfarb to make this code a little easier to read you can use template literals here:

```js
  let cityIcon = `
    <svg 
      width="${2.4 * pointRadius}" 
      height="${2.4 * pointRadius}"
    >
      <circle 
        cx="${1.2 * pointRadius}" 
        cy="${1.2 * pointRadius}" 
        r="${pointRadius}"
        stroke="#4dac26"
        stroke-width="${strokeWeight}"
        fill="#b8e186"
        fill-opacity="${fillOpacity}"
      />
    </svg>`;
```

Reply:
we may also want to have variables for the hexcodes for the various strokes and fills instead of copying them manually. There are other variables for the layer styling on line 30, so we could add them there.